modules/core/shared/src/main/scala/scaladex/core/api/artifact/ArtifactMetadataResponse.scala

```diff
@@ -0,0 +1,9 @@
+package scaladex.core.api.artifact
+
+final case class ArtifactMetadataResponse(
+    version: String,
+    projectReference: Option[String],
```

Comment:
This field in practice is always `Some`, is there a reason why you want it to be `None`?

I think there's work planned [in #869](https://github.com/scalacenter/scaladex/issues/869) that would index artifacts that don't have an associated project – are we just planning for this?

Reply:
You are 100% right. Currently the field can only be `Some` and we plan that it can be `None` in #869.